lib/rubocop/cop/mixin/empty_lines_around_body.rb

```diff
@@ -21,14 +21,11 @@ module EmptyLinesAroundBody
         def_node_matcher :constant_definition?, '{class module}'
         def_node_matcher :empty_line_required?, '{def defs class module}'
 
-        def check(node, body)
-          # When style is `empty_lines`, if the body is empty, we don't enforce
-          # the presence OR absence of an empty line
-          # But if style is `no_empty_lines`, there must not be an empty line
-          return unless body || style == :no_empty_lines
+        def check(node, body, adjusted_first_line: nil)
```

Comment:
Do we have an official position on “complex” keyword arguments? Because this method signature *could* be written as such:
```rb
def check(node, body, first_line: node.source_range.first_line)
```
and avoid the `nil`-check in line 28.

Reply:
AFAIK, unfortunately it does not behave as expected. This is because the default value of keyword argument is overwritten with `nil` when the argument is given `nil`.

```ruby
def say(message: 'hello')
  puts message
end

say                # => message is 'hello'
say(message: 'hi') # => message is 'hi'
say(message: nil)  # => message is nil
```

Reply:
Yes, that is true. One would need to be careful not to pass `nil`.